js/flightlog_fielddefs.js

```diff
@@ -349,6 +349,19 @@ let
         "ATTITUDE",
         "VTX_MSP",
         "GPS_DOP",
+        "FAILSAFE",
+        "GYRO_CALIBRATION",
+        "ANGLE_MODE",
+        "ANGLE_TARGET",
+        "CURRENT_ANGLE",
```

Comment:
Missing debug modes should be added in separate PR as probably needs to be added using `adjustFieldDefsList` function

Reply:
resolved in current codebase